client/test/role.go

```diff
@@ -284,6 +292,43 @@ func (r *role) ProposeChannel(req client.ChannelProposal) (*paymentChannel, erro
 	return ch, nil
 }
 
+func (r *role) RequireNoError(err error) {
```

Comment:
As written above, it would be possible to call `runtime.GoExit()` in those `Require` methods, which will lead to stopping the go-routine in which they are called.